ui/src/app/pages/data/cohort/modifier-page.tsx

```diff
@@ -514,7 +514,6 @@ export const ModifierPage = fp.flow(
 
     const calculate = async () => {
       const { domain, role } = cohortContext;
-      const { id, namespace } = workspace;
```

Comment:
don't need this because it already has the matchparams.

Renaming both the params and the workspace entity field to `terraName` introduced a shadowing condition which made it an error to include both.